runner/execer/execers/sim.go

```diff
@@ -11,24 +11,28 @@ import (
 	"github.com/scootdev/scoot/runner/execer"
 )
 
-func NewSimExecer(wait *sync.WaitGroup) execer.Execer {
-	return &simExecer{wait}
+func NewSimExecer() *SimExecer {
+	return &SimExecer{resumeCh: make(chan struct{})}
 }
 
-type simExecer struct {
-	wait *sync.WaitGroup
+type SimExecer struct {
+	resumeCh chan struct{}
 }
 
-// simExecer execs by simulating running argv.
+// SimExecer execs by simulating running argv.
 // each arg in command.argv is simulated in order.
 // valid args are:
 // complete <exitcode int>
 //   complete with exitcode
 // pause
-//   pause until the simExecer's WaitGroup is Done
+//   pause until the SimExecer's WaitGroup is Done
```

Comment:
I think this comment needs to be changed?  (// pause until the SimExecer's Abort() or Resume() is called.)?